packages/ag-charts-community/src/chart/series/cartesian/lineSeries.ts

```diff
@@ -59,6 +59,26 @@ class LineSeriesLabel extends Label {
 type LineContext = SeriesNodeDataContext<LineNodeDatum>;
 type LineAnimationData = CartesianAnimationData<LineContext>;
 
+// TODO(olegat) move somewhere else?
+export class ErrorBarDatum {
+    @Validate(OPT_STRING)
+    yLowerKey: string;
```

Comment:
A quirk/smell of TS->JS is that if a field is not assigned a value, it doesn't exist at runtime - typically you'll see that fields that are `@Validate()`'ed also assign an initial value, even if that is `undefined` - this allows us to sanity check that options provided by users actually exist before assignment from the users options, rather than just blinding adding.

Reply:
Thanks for the explanation. That makes sense. What values should be used?

I'm thinking of using an empty string `''` for `yLowerKey` and `yUpperKey` because these properties are required, and `undefined` for `yLowerName` and `yUpperName` because these are option.